app/utils/Configuration.scala

```diff
@@ -28,7 +28,8 @@ class Configuration(val bindPort: Int = ConfigFactory.load().getInt("app.portWeb
   val instanceName = "WebAppInstance"
   val instanceRegistryUri : String = sys.env.getOrElse("DELPHI_IR_URI", ConfigFactory.load().getString("instance.registry.path"))
 
-  lazy val webApiUri:String =  webApiInstance.host + ":" + webApiInstance.portNumber
+  lazy val webApiUri:String =  webApiInstance.host
```

Comment:
By the way... Here was the original base URL retrieved through the instanceregistry or the default configuration.

Reply:
Issues have been addressed

Reply:
kind of... :)